File: java/Problem1043.java
import java.util.Scanner;

public class Problem1043 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double x = sc.nextDouble();
        double y = sc.nextDouble();
        double z = sc.nextDouble();

        if((x + y) <= z || (x + z) <= y || (y + z) <= x){
            double area = ((x + y)*z)/2;
            System.out.printf("Area = %.1f\n", area);
        } else {
            System.out.printf("Perimetro = %.1f\n", (x + y + z));
        }

        sc.close();
    }
}

File: python/problem1043.py
valores =  list(map(float, input().split()));

a = valores[0];
b = valores[1];
c = valores[2];

if a + b > c and a + c > b and c + b > a:
	perimetro = a + b + c;
	print("Perimetro = %.1f" % perimetro);
else:
	area = 0.5 * (a + b) * c;
	print("Area = %.1f" % area);
